tests/console-backend-service/internal/domain/cms/clusterdocstopic_test.go

```diff
@@ -44,12 +45,17 @@ func TestClusterDocsTopicsQueries(t *testing.T) {
 	cmsCli, _, err := client.NewDynamicClientWithConfig()
 	require.NoError(t, err)
 
+	t.Log("Setup test service")
+	host, err := mockice.Start(cmsCli, MockiceNamespace, MockiceSvcName, 8080)
```

Comment:
Please extract port to constant, because I see that you also use it same value in `service-catalog` test.

Reply:
Moved port to `mockice` package.